enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/EnterpriseCoreEditionModule.java

```diff
@@ -201,6 +211,15 @@ public void registerEditionSpecificProcedures( Procedures procedures ) throws Ke
         life.add( coreServerModule.membershipWaiterLifecycle );
     }
 
+    static Predicate<String> fileWatcherFileNameFilter()
+    {
+        return Predicates.any(
+                fileName -> fileName.startsWith( PhysicalLogFile.DEFAULT_NAME ),
+                fileName -> fileName.startsWith( IndexConfigStore.INDEX_DB_FILE_NAME ),
+                filename -> filename.startsWith( StoreUtil.TEMP_COPY_DIRECTORY_NAME )
```

Comment:
we are watching for a directory content: our graph db directory and it's parent to see when graph db directory will be removed by itself. We can't just watch for specific files. That's why we need to supply list of exception filters that we will use to filter out file names that we should not notify about.
That's we we need to pass `temp-copy` to filter out all possible notification about it that can come alter.